notary-api/src/operations/notaryInfo.js

```diff
@@ -0,0 +1,18 @@
+import config from '../../config';
+import signingService from '../services/signingService';
+
+export const getSignedNotaryInfo = async () => {
+  try {
+    const { notaryName, notaryPublicBaseUrl } = config;
+    const info = {
+      name: notaryName,
+      notarizationUrl: `${notaryPublicBaseUrl}/buyers/notarization-request`,
+      dataResponsesUrl: `${notaryPublicBaseUrl}/data-responses`,
+      headsUpUrl: `${notaryPublicBaseUrl}/sellers/heads-up`,
+    };
+
+    return signingService.signNotaryInfo(info);
```

Comment:
The operation needs to wait for the response to catch and return the error. Otherwise, it will end in a unhandled promise rejection:
```suggestion
    const response = await signingService.signNotaryInfo(info);
    return response;
```

Reply:
Also you'll (almost) never hit the `catch` block, since (almost, again) nothing could fail in the `try` block

Reply:
Yes! The request to the signing service!